source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -404,7 +404,8 @@ message Transformation {
     TransformationAddSynonym add_synonym = 57;
     TransformationAddRelaxedDecoration add_relaxed_decoration = 58;
     TransformationReplaceParamsWithStruct replace_params_with_struct = 59;
-    TransformationReplaceCopyObjectWithStoreLoad replace_copy_object_with_store_load = 60;
+    TransformationReplaceCopyMemoryWithLoadStore replace_copy_memory_with_load_store = 60;
+    TransformationReplaceCopyObjectWithStoreLoad replace_copy_object_with_store_load = 61;
```

Comment:
When you rebase on top of the change I just merged, keep the copy object transformation as 60 and make this one 61.